feat: update doc and changelog about APIM upgrade installation on redhat (#1128)
gravitee-io/gravitee-docs: pages/apim/3.x/installation-guide/red-hat/installation-guide-redhat-stack.adoc

curl -L https://bit.ly/install-apim-3x | bash
----

=== Upgrade APIM

> In order to upgrade your APIM installation, you will have to proceed the package upgrade and then restart apim:
> 
> NOTE: Since maintained version 3.20.24 and above the rpm upgrade of APIM component is easier. Now you can simply upgrade and restart APIM.

Reviewer: ```suggestion
As of APIM 3.20.24, to upgrade an APIM component you can simply do a `yum upgrade` and restart APIM:
```

> WARNING: Always take a look on the changelog to follow potential breaking change.

Reviewer: ```suggestion
WARNING: Always look at the changelog to follow potential breaking changes.
```
